src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -348,7 +377,7 @@ public void resetOdometry(Pose2d startPose, Rotation2d startGyro) {
   }
 
   public void updateOdometry() {
-    m_pose = m_odometry.update(Rotation2d.fromDegrees(-m_sensors.getYaw()), Units.feetToMeters(getLeftPosition()), Units.feetToMeters(getRightPosition()));
+    m_pose = m_odometry.update(Rotation2d.fromDegrees(m_sensors.getYaw()), Units.feetToMeters(getLeftPosition()), Units.feetToMeters(getRightPosition()));
```

Comment:
@nix342 I want to delete the `-` here. We have the `getYaw()` method setup to return positive=counter-clockwise, which is the standard in WPILib, including the simulation stuff. This might break our autonomous, though, so we need to watch out for that

Reply:
I've tested and things seems to be working correctly in the sim with trajectory following.

I'm going to accept this PR. I've got a new one incoming with trajectory stuff.